### ISSUE TITLE
Imbalanced and small quantity dataset.Which model can I try?

### ISSUE THREAD

**Issue by lyy-fighting** [open-mmlab/mmdetection#2873]
My custom dataset has 90 categories and 1300 images in which half number is nature images,the rest specimen.The dataset is imbalanced among classes. I havn't augmented each image 10 times to create 13000 images. I split train and val set with ratio 8:2. I had a try to employ retinanet with resnet(18,50,101)and ssd300.But overfitting appears so that train set performs well but validation set is bad . How can I do my experiment? Which model can I try? Please help me if you have solved similar situation ,Thanks!

**Comment by hubutui** [comment]
why not try faster rcnn first?

**Comment by jefflgaol** [comment]
Did you monitor your loss graph to see whether you need to perform early stopping?

**Comment by analyticsbot** [comment]
Hi @lyy-fighting - were you able to get good performance from the repo for imbalanced classes?
